### PR TITLE
TASK 5 implement get articles(all)

### PR DESCRIPTION
**Pull request by YShamm**

### CODE REVIEW

File: playground.sql [hunk 1]
```diff
@@ -1,3 +1,7 @@
 \c nc_news_test
 
-SELECT * FROM articles WHERE article_id =3;
\ No newline at end of file
+SELECT * FROM comments;
+SELECT * FROM articles;
+
+SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC; 
+-- ORDER BY created_at DESC;
```
Comment (Harrison365) [inline playground.sql:7]: Nice 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -97,3 +97,28 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+        });
```
Comment (Harrison365) [inline __tests__/app.test.js:121]: We also want to check the type of the property values. Have a look into 
```
expect.objectContaining({
            body: expect.any(String),
```

**Comment by Harrison365** [comment]
Just a small comment in the tests. :) 
Very good work ✅

**Review by Poonam-raj — APPROVED** [review]
Well done Yahya, i would definitely remove your console logs comments before committing.

Feel free to merge

**Pull request merged**